pkgs/top-level/all-packages.nix

```diff
@@ -28939,4 +28939,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  unused = callPackage ../development/tools/unused {};
```

Comment:
Please do not put this at the end of the package list but sort it in a fitting category and in that alphabetically.

Reply:
I didn't notice there were categories. Fixed.